drools-osgi/drools-karaf-itests/README

```diff
@@ -4,6 +4,8 @@ These additional properties can be used to execute tests against custom karaf
 or karaf like container (for example JBoss FUSE). 
 
 karaf.dist.file - path to karaf distribution file (if not defined, the default karaf is downloaded from maven central)
+karafVersion - Version of Karaf container. This parameter is mandatory when a custom Karaf distribution file is specified.
```

Comment:
We could maybe use `karaf.version`, so that the name is consistent with other properties. WDYT?